packages/one-app-hmr/lib/index.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+// eslint-disable-next-line no-global-assign
+require = require('esm')(module, { mainFields: ['module', 'main'] });
```

Comment:
This allows us to use `import`/`export` syntax directly in the `lib/`; future proof since `node.js` 14 does support `"type": "module"` in `package.json`.